### PR TITLE
Suicide Prevention Week

### PR DESCRIPTION
**Pull request by Squird5**
90 on the highway is a lifestyle choice

## About The Pull Request

I made some minor tweaks to Specials, due to some conversation about them several days ago.

* Specials now persist through the entire round, so people don't "reroll" them, as per the Title.
* Added a few curses as Specials
  * Reworked some of the systems to make that possible
  * Reworked Baotha's Curse to be less screen flashing and hellish, added arousal instead
  * Added the hook for Baotha's Curse to the Love Potion, since that was theorized in the previously mentioned conversation
  * Added "cumbrained" debuff for Baotha's Curse
* Someone fucking removed the LowMemory compile option, so fuck them it's back <sub>(commented out, of course (stop fucking staging files that aren't part of your fucking PR))</sub>

## Why It's Good For The Game

More things are more good :^)

**I spent the entire day testing this over and over and over again until I made sure it worked.** I don't have any screenshots or videos because what are you going to show off?

### CODE REVIEW

**Comment by babamandaa** [comment]
I _really_ like cumbrained, but I will say for the record that some specials kind of just disable certain playstyles and I don't like having _no_ way to get rid of a bad special across an entire round. Maybe completely disable Lazy and make it a special you can opt into in place of the special you rolled? That way, if you want to play like bandit or something but you roll pacifist you're not completely 100% screwed over. Still sucks, but it doesn't bar you from doing anything.

**Comment by katsukai** [comment]
Don't approve making gourmand most common special to roll at 200 weight. Makes food non consequential.

**Comment by Heheniy** [comment]
you said nothing about changing traits yet you did that

File: code/datums/special_traits/traits/traits.dm [hunk 1]
```diff
@@ -483,6 +482,31 @@
...
+	name = "Nymphomaniac"
+	greet_text = span_boldwarning("I'm in a constant state of arousal, and I cannot control my urges.")
+	weight = 50
+
+/datum/special_trait/nymphomaniac/on_apply(mob/living/carbon/human/character, silent)
+	character.add_curse(/datum/curse/baotha, TRUE)
+	ADD_TRAIT(character, TRAIT_GOODLOVER, "[type]")
+
+/datum/special_trait/loveless
+	name = "Nymphomaniac"
```
Comment (Gyrandola) [inline code/datums/special_traits/traits/traits.dm:495]:
```suggestion
	name = "Loveless"
```

**Comment by Nakalon** [comment]
Reading sayonaras comment... Maybe it's best this system isn't fiddled with. I'm theory it's good but I don't know if it will have the intended effect...

File: code/datums/special_traits/traits/traits.dm [hunk 2]
```diff
@@ -48,7 +44,7 @@
 /datum/special_trait/value
 	name = "Coin Counter"
 	greet_text = span_notice("I know how to estimate an item's value.")
-	weight = 100
+	weight = 50
```
Comment (atomicplan666) [inline code/datums/special_traits/traits/traits.dm:47]: Or this

File: code/datums/special_traits/traits/traits.dm [hunk 3]
```diff
@@ -57,23 +53,23 @@
 /datum/special_trait/lightstep
 	name = "Light Step"
 	greet_text = span_notice("I am quiet, nobody can hear my steps.")
-	weight = 100
+	weight = 150
```
Comment (atomicplan666) [inline code/datums/special_traits/traits/traits.dm:56]: Or this

**Comment by Squird5** [comment]
> if you want to play like bandit or something but you roll pacifist you're not completely 100% screwed over. Still sucks, but it doesn't bar you from doing anything.

1_-Z4PlTwt6tqp6ZBaUjorAQ

> you said nothing about changing traits yet you did that

What?

> What if I got the trait "Immortality", I was killed as bog guard and then joined in as another character with the same trait?
> 
> I don't think that really fair though. You should think about positive traits not only for negative

Sure, but not all of them will give you the trait if it's in a restricted class.
The idea is for people to stop round start suiciding because they got "a bad roll". You rolled, knowing full well you could get something shit. Deal with it. Don't make a random name cheese character to go neck yourself at the dock. Fucking play it and roleplay with it.

**Comment by Squird5** [comment]
Persistent special is a single line change at least. I'd like to see how it goes locked in as this intends, but if it does not go well, that line can be easily readded.

File: code/modules/research/xenobiology/crossbreeding/_potions.dm [hunk 4]
```diff
@@ -73,7 +73,7 @@ Slimecrossing Potions
 	icon = 'icons/roguetown/items/cooking.dmi'
 	icon_state = "lovebottle"
 
-/obj/item/slimepotion/lovepotion/attack(mob/living/M, mob/user)
```
Comment (Cyprex) [inline code/modules/research/xenobiology/crossbreeding/_potions.dm:76]: No? This handles only living things. What you declare here is merely a suggestion.

**Comment by katsukai** [comment]
> why are you doing all those changes to existing traits? you explained nothing this is not even a "minor tweak" image

I designed that special to be a minor buff with decent weight. Not an OP buff to be stronger than giant.

**Comment by Nakalon** [comment]
> Persistent special is a single line change at least. I'd like to see how it goes locked in as this intends, but if it does not go well, that line can be easily readded.

Persistent special is the only thing I'm seriously considering in this PR

**Comment by Gyrandola** [comment]
This PR needs to be atomized

**Pull request closed without merging**